src/views/open_order_list_view.js

```diff
@@ -0,0 +1,167 @@
+import Backbone from 'backbone';
+import OpenOrderView from '../views/open_order_view'
+
+import OpenOrder from '../models/open_order';
+
+const OpenOrderListView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.bus = params.bus;
+    this.listenTo(this.model, 'update', this.render);
+    this.listenTo(this.bus, 'quote_symbols', this.symbolDropdown);
+    this.listenTo(this.bus, 'quote_change', this.checkOpenOrders);
+
+    // list for current quote list
+    this.listenTo(this.bus, 'current_quote_list', this.getQuoteList);
+  },
+  getQuoteList(quoteList) {
+    this.quoteList = quoteList;
+    console.log('the quote list !!')
+    console.log(this.quoteList);
+
+  },
+  render() {
+    this.$('#orders').empty();
+
+    this.model.each((openOrder) => {
+      const openOrderView = new OpenOrderView({
+        model: openOrder,
+        template: this.template,
+        tagName: 'li',
+        className: 'order',
+        bus: this.bus,
+      });
+      // render returns the taskview (this) which allows you to append
+      this.$('#orders').append(openOrderView.render().$el);
+    });
+    return this;
+  },
+  checkOpenOrders(quote) {
+
+    let openOrders = this.model.where({symbol: quote.get('symbol')});
```

Comment:
Currently, the workflow for executing an open order looks like this:

1. Quote price changes
1. `quote_change` event on the bus
1. Handled here by `OpenOrderListView.checkOpenOrders()`
1. Find `OpenOrder`s matching the quote
1. For each one, determine if it needs to be sold

But we can simplify this! The key observation is that each `OpenOrder` already knows which `Quote` it's for - we need this information to validate the `OpenOrder`. Instead of working through the intermediaries of the bus and the `OpenOrderListView`, each `OpenOrder` could listen directly to its `Quote` for price changes. Then the workflow would look like:

1. Quote price changes
1. Event on the quote
1. Handler in `OpenOrder`
1. Determine if the order needs to be sold

This eliminates dependencies on the bus and the `OpenOrderListView`. It also simplifies the code, since we don't need to work with a list of `OpenOrder`s. If there are multiple `OpenOrders` corresponding to the same quote each will have a separate handler registered, and each will run when the event occurs (steps 3-4).